docs/concepts/timely-stream-processing.md

```diff
@@ -24,17 +24,25 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-`TODO: add introduction`
-
 * This will be replaced by the TOC
 {:toc}
 
+## Introduction
+
+Timely steam processing is an extension of [stateful stream processing]({% link
+concepts/stateful-stream-processing.md %}) in which time plays some role in the
+computation. Among other things, this is the case when you do time series
+analysis, when doing aggregations based on certain time periods (typically
+called windows), or when you do event processing where the time when an event
+occured is important.
+
+In the following sections we will highlight some of the topics that you should
+consider when working with timely Flink Applications.
+
```

Comment:
Putting the toc after the intro is more consistent with the rest of the docs. 

```suggestion
Timely steam processing is an extension of [stateful stream processing]({% link
concepts/stateful-stream-processing.md %}) in which time plays some role in the
computation. Among other things, this is the case when you do time series
analysis, when doing aggregations based on certain time periods (typically
called windows), or when you do event processing where the time when an event
occurred is important.

In the following sections, we will highlight some of the topics that you should
consider when working with timely Flink Applications.

* This will be replaced by the TOC
{:toc}
```

Reply:
I moved it after the toc because @StephanEwen recently mentioned that he puts the toc first, without an intro before. We should decide on one way of doing it and stick to it. What do you two think?